xExtension-StickyFeeds/README.md

```diff
@@ -1,5 +1,9 @@
-# Sticky Feeds extension
+# Sticky Feeds extension (Depricated)
```

Comment:
deprecated :-)

But I think all files except for this one should be removed, leaving just this README file, also stripped of at least that last line.

Reply:
Also to be removed (you could add it to this PR if you like)
https://github.com/FreshRSS/Extensions/blob/e69e454618dd63bece0e275832ad984799f59bc3/extensions.json#L28-L35